package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "graphql-yoga",
   "version": "0.0.0-semantic-release",
-  "main": "dist/src/index.js",
-  "typings": "./dist/src/index.d.ts",
+  "main": "dist/index.js",
+  "typings": "./dist/index.d.ts",
```

Comment:
Is `./` necessary here?

Reply:
🤷‍♂️  

I didn't add it. :)

Reply:
I’m wondering if main and typings paths could be brought to the same format. This PR looks like a good opportunity to do so.

GraphQL yoga may be a popular thing for TS newcomers to inspect and so it’d be great if it’s code was clean and did not raise unnecessary questions.

Reply:
Updated PR